codegen/object.go

```diff
@@ -84,8 +84,10 @@ func (b *builder) buildObject(typ *ast.Definition) (*Object, error) {
 func (o *Object) Reference() types.Type {
 	switch v := o.Type.(type) {
 	case *types.Named:
-		_, isInterface := v.Underlying().(*types.Interface)
-		if isInterface {
+		if _, isInterface := v.Underlying().(*types.Interface); isInterface {
```

Comment:
I think this should be calling https://github.com/99designs/gqlgen/blob/14dccc57885df5d5ca4ef347c1b80f5f3648719a/codegen/config/binder.go#L413 (and make it public) to stop these falling out of sync again in the future?